libvirt.go

```diff
@@ -153,6 +153,31 @@ const (
 	DestroyFlagGraceful
 )
 
+// DomainState specifies state of the domain
+type DomainState uint32
+
+const (
+	// DomainStateNoState No state
+	DomainStateNoState = 0 << iota
```

Comment:
`libvirt-domain.h` has these in sequential order:
```c
typedef enum {
    VIR_DOMAIN_NOSTATE = 0,     /* no state */
    VIR_DOMAIN_RUNNING = 1,     /* the domain is running */
    VIR_DOMAIN_BLOCKED = 2,     /* the domain is blocked on resource */
    VIR_DOMAIN_PAUSED  = 3,     /* the domain is paused by user */
    VIR_DOMAIN_SHUTDOWN= 4,     /* the domain is being shut down */
    VIR_DOMAIN_SHUTOFF = 5,     /* the domain is shut off */
    VIR_DOMAIN_CRASHED = 6,     /* the domain is crashed */
    VIR_DOMAIN_PMSUSPENDED = 7, /* the domain is suspended by guest
                                   power management */
...
```
So there's no need for a bit-shift here, this can be simplified to:
```go
DomainStateNoState = iota
```